src/main/java/uk/gov/companieshouse/web/accounts/controller/smallfull/ApprovalController.java

```diff
@@ -28,13 +29,14 @@ public String getApproval() {
     @PostMapping
     public String postApproval(@PathVariable String companyNumber,
                                @PathVariable String transactionId,
-                               @PathVariable String companyAccountsId) {
+                               @PathVariable String companyAccountsId,
+                               HttpServletRequest request) {
 
         try {
             transactionService.closeTransaction(transactionId);
         } catch (ApiErrorResponseException e) {
             // TODO: handle ApiErrorResponseExceptions (SFA-594)
-            LOGGER.error(e);
+            LOGGER.errorRequest(request, "Failed to close transaction");
```

Comment:
We should be passing the exception to the logger as it may contain extra details that help us identify why the service "Failed to close transaction"
This might require an extra method being exposed in the structured-logging library to allow an exception and message at the same time but thats better than losing the specific details of the exception (stack trace and cause exception)